docs/book/src/developer/release.md

```diff
@@ -32,7 +32,7 @@
     ```
 - Wait for the google cloud build to be finished
 - Create a draft release with release notes for the tag
-- Perform the [image promotion process](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io#image-promoter):
+- Perform the [image promotion process](https://github.com/kubernetes/k8s.io/tree/main/registry.k8s.io#image-promoter):
```

Comment:
this doesn't need the change, lets stick to the original link.

Reply:
done，please review